### PR TITLE
docs: add SQLite support

### PR DESCRIPTION
**Pull request by laysakura**
PR to #762

### CODE REVIEW

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you!

File: docs/scalardb-supported-databases.md [hunk 1]
```diff
@@ -10,21 +10,23 @@ Scalar DB supports the following databases and the databases that are compatible
...
-| Scalar DB 2.0 |  ✅  |  ✅  |  ✅  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |
+| Scalar DB Version | Cassandra 3.11 | Cassandra 3.0    | Cassandra 2.2   | Cosmos DB | DynamoDB  | MySQL 8.0  | MySQL 5.7  | PostgreSQL 14 | PostgreSQL 13 | PostgreSQL 12 | Oracle 21.3.0-xe   | Oracle 18.4.0-xe | SQL Server 2022 | SQL Server 2019 | SQL Server 2017 | AWS Aurora MySQL 3 (MySQL 8) | AWS Aurora MySQL 2 (MySQL 5.7) |  AWS Aurora PostgreSQL 14 |  AWS Aurora PostgreSQL 13 | AWS Aurora PostgreSQL 12 | SQLite 3 |
+|:------------------|:---------------|:-----------------|:----------------|:----------|:----------|:-----------|:-----------|:--------------|:--------------|:--------------|:-------------------|:-----------------|:----------------|:----------------|:----------------|:-----------------------------|:-------------------------------|:--------------------------|:--------------------------|:-------------------------|:---------|
+| Scalar DB 3.8 |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |
```
Comment (brfrn169) [inline docs/scalardb-supported-databases.md:19]: Our QA team will update it after releasing Scalar DB 3.8 and testing the compatibilities. Can you please revert this change? I think you can just add a column for SQLite to this table. Thanks.

**Comment by brfrn169** [comment]
@feeblefakie @komamitsu Can you please take a look this PR, it's already merged though.

File: docs/scalardb-supported-databases.md (hunk 1 above)
Comment (laysakura) [reply docs/scalardb-supported-databases.md:19]: Sure. I got it.

Comment (laysakura) [reply docs/scalardb-supported-databases.md:19]: https://github.com/scalar-labs/scalardb/pull/769

**Review by komamitsu — COMMENTED** [review]
LGTM! 👍
